### ISSUE TITLE
[BUG] Switch Pro Not detected, but detected a phantom Joy-con(R)

### ISSUE THREAD

**Issue by Okanegahoshi** [Davidobot/BetterJoy#751]
**Describe the bug**
I used the version 6.3, because both 6.4a and 6.4b refuses to launch the window. (I know the version is supposed to fix it but it just doesn't.)
Anyway, my Pro controller is wired connected, Windows Device Manager recognizes it, Steam recognizes it, but BetterJoy doesn't.
What's weirder, it says I have a Right Joy-con registered, but Windows Device Manager doesn't, and my real Joy-Con on the table doesn't seem to be connected to anything.
I tried closed the Steam and launch BetterJoy again, nothing changes.

(Side notes: There might be issue on my desktop and pro controller, DS4Windows can't register it either. My controller can wirelessly connect to my laptop, but failed when wired; My desktop Bluetooth doesn't work, but wired works...but all of these wasn't the case before I switched to Smash Style Pro controller to replace the default skin one despite they should be functionally identical. I have no idea what's going on. Maybe Steam is the exception to handle it properly.)

**Expected behavior**
I connected the controller, the software says the controller is connected without phantoms, and I'm able to play BotW on PC with controller, I guess?

**Controller (please complete the following information):**
 - Type: Switch Pro Controller (Smash Bros.)
 - Connection: USB (BT doesn't work on hardware level.)

**Comment by Okanegahoshi** [comment]
Okay, this is pretty weird, but let me explain...
So it turns out, my BT didn't work isn't a hardware issue, but a driver one.
I installed something called Bluesoleil ages ago, I never realized it until I tried a new BT receiver today. 
What is Bluesoleil? Idk, but It demand a license purchase to open access to "full feature set", whatever it is.
And. If I just uninstall that, the BT can now be used by Windows itself. Which is able to normally connect the controller.
Also the phantom is gone. I supposed that shady driver also sent false signals for whatever reason.
And the weirdest part is, connecting via USB works now. 
Just how much did that thing messed up my computer for years, I have no idea.
But everything works out in the end. How fabulous.

TL;DR: It's all because of a shady BT driver called Bluesoleil. Uninstalled = Problem solved.
